add piggyback toggle and banning

6fears7 opened the pull request:
<img width="539" height="557" alt="image" src="https://github.com/user-attachments/assets/c5115550-a0b2-4f47-87aa-3d54aa731689" />

https://github.com/user-attachments/assets/843f6d13-6da1-4400-b8fb-abe2c66887b7

6fears7 committed:
add piggyback toggle

6fears7 committed:
piggy back override

Dustylemny commented:
> 1. The slug button supports description like that
<img width="908" height="79" alt="image" src="https://github.com/user-attachments/assets/278c9e55-0817-46a3-83ab-35e102c32aaf" />
It should, you can check the id matches "scug select" above in ArenaOnlineLobbyMenu.UpdateInfoText() and put the description there.

Im okay with having this but however the main thing is that does it add itself when you become a new host for the lobby?

Dustylemny committed:
- Add ability to press grab while hovering over slugcat button to toggle ban
- Filter slugcats that the player can play as for playing as Random (unless random slugcat is the only available slugcat)
- Change ur slugcat if its not available
- Small fix where you cant toggle arena game modes using movement keybinds

Dustylemny committed:
Rename Piggy config and reposition Better Hitbox config

Dustylemny committed:
Check for unbannedIndex for changing slugcat with shift

6fears7 committed:
Merge pull request #1021 from Dustylemny/arena-banslugs-usinggrab

Toggle Arena Slugcat Banning using Grab

Dustylemny committed:
Make banned slugcat illustrations set to dead

Dustylemny committed:
Remove redundant line

6fears7 committed:
Merge pull request #1024 from Dustylemny/arena-showbannedslugdead

Make banned slugcat illustrations set to dead
